client/src/pages/ResetPassword/ResetPassword/PasswordForm/PasswordForm.tsx

```diff
@@ -0,0 +1,67 @@
+import { Typography, TextField, Box, Grid, Button, CircularProgress } from '@material-ui/core';
+import { FormikHelpers, Formik } from 'formik';
+import useStyles from './useStyles';
+import * as Yup from 'yup';
+interface Props {
+  handleSubmit: (
+    {
+      password,
+    }: {
+      password: string;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      password: string;
+    }>,
+  ) => void;
+}
+
+export default function EmailForm({ handleSubmit }: Props): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Formik
+      initialValues={{
+        password: '',
+      }}
+      validationSchema={Yup.object().shape({
+        password: Yup.string()
+          .required('Password is required')
+          .max(100, 'Password is too long')
+          .min(6, 'Password too short'),
+      })}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <label htmlFor="password">{<Typography className={classes.label}>Password</Typography>}</label>
```

Comment:
You can use the MUI label component here to stay consistent with the design and code base

Ref: https://material-ui.com/api/form-label/

Reply:
thanks for the reference, changed it to use FormLabel  component